content/en/gitops.md

```diff
@@ -1,31 +1,25 @@
 ---
 title: GitOps
-status: Feedback Appreciated
+status: Completed
 category: Concept
-tags: ["", "", ""]
+tags: ["methodology"]
 ---
 
 ## What it is
 
-GitOps is a set of best practices based on [shared principles](https://opengitops.dev/), 
-applied to a workflow that depends on software agents that 
-enable automation to reconcile a declared system state or configuration in a git repository.
-These software agents and practices are used to execute a cohesive workflow that 
-leverages a source control system like Git as the “single source of truth” and 
-extends this practice to applications, infrastructure, and operational procedures.
+GitOps is a set of best practices based on [shared principles](https://github.com/open-gitops/documents/blob/release-v1.0.0/PRINCIPLES.md), applied to a workflow that depends on software agents to continuously and automatically reconcile an actual running system with its desired state declared in an immutably versioned state store like Git.
+These software agents and practices are used to execute a cohesive workflow and extends this practice to applications, infrastructure, and operational procedures.
 
 ## Problem it addresses
 
-Existing processes for infrastructure configuration management can face challenges 
-such as configuration drift, failed deployments, relying on a system's previous state for success, 
-missing documentation, or unknown development history.
+Existing processes for infrastructure configuration management – including scripting and tightly coupled CI/CD – are often error prone and can face challenges such as configuration drift, failed deployments, relying on a system's previous state for success, missing documentation, or unknown development history.
+Event driven CD does not address drift that may happen independently of CI.
+Giving external automation tools credentials to your production systems widens the attack footprint.
 Adopting a GitOps workflow can help alleviate these issues, among several others.
 
 ## How it helps
 
-GitOps is a paradigm that can be applied to a workflow 
-to help manage an application and cloud system infrastructure. 
-It enables organizations several advantages 
-such as better coordination, transparency, stability, and reliability of a system.
-Operating in a close loop ensures the current live state of a system matches 
-against the desired target state, specified in the git repository.
+The GitOps paradigm can be applied to a workflow to help manage applications and cloud system infrastructure. 
+It enables organizations several advantages such as better coordination, transparency, stability, and reliability of a system, and faster mean time to recovery.
+Direct access to desired state store and continuous reconciliation allows GitOps agents to ensure the current live state of a system matches the desired target state, regardless of when or how any system failure or drift between the two may occur.
```

Comment:
Some of what is here, is captured in the Google doc but in simpler language. Can repurpose some of this?

> Traditionally seen as a single process, GitOps decouples CI/CD (principles 3 and 4). While [CI](https://glossary.cncf.io/continuous-integration/) is focused on building applications iteratively, CD focuses on seeing that change in the target system. GitOps concentrates on the CD aspect — both [delivery](https://glossary.cncf.io/continuous-delivery/) and [deployment](https://glossary.cncf.io/continuous-deployment/). That allows developers to focus on CI without worrying about how the change ends up on the target system — they just need to know that it does.
> Configuration files, stored as code in a repository ([infrastructure as code](https://glossary.cncf.io/infrastructure-as-code/)), generate the same infrastructure environment each time it is deployed, creating consistency. And to ensure what's running on the cluster doesn't diverge from the desired state (codified in Git), [software agents](https://www.britannica.com/technology/software-agent) continuously monitor the live environment. The agent will attempt to roll back or match the desired state if divergence is detected, avoiding configuration drift.